packages/gestalt/src/IconButton.js

```diff
@@ -87,7 +87,7 @@ const IconButtonWithForwardRef: React$AbstractComponent<unionProps, unionRefs> =
     dangerouslySetSvgPath,
     disabled,
     icon,
-    iconColor,
+    iconColor = 'darkGray',
```

Comment:
Huh…we weren't setting the default color before? That makes me suspicious that it's still being defined somewhere…have you looked into that?

Reply:
Pog sets "gray" based on the default bgColor which is transparent.